### ISSUE TITLE
can't cast to chromecast 

### ISSUE THREAD

**Issue by gcadoret** [LagradOst/CloudStream-3#383]
hello
I'm using last version 2.4.7 and when I start to cast, it seems to work but after 2s, the player stops. a little difficult to explain. any way to see the log?

**Comment by GeorgeLengyel** [comment]
Same thing is happening to me. Stream seems to switch to a different source within a few seconds when Chromecasting and doesn't ever load. Any ideas?

**Comment by g-meister** [comment]
![image](https://user-images.githubusercontent.com/75985414/147985282-004d2118-4229-4675-b8d5-38ed8e0f5c0b.png)
Had a look into the issue, seems the media playback is reporting LOAD_FAILED. 
https://developers.google.com/cast/docs/web_receiver/error_codes

Note: tested with trying Dopebox -> Skyfall

**Comment by Curve** [comment]
Can confirm, app works great when sideloaded tho.

**Comment by transferableethos** [comment]
Is it working for you now? I've been trying to cast and it isn't working.

**Comment by transferableethos** [comment]
> no, it doesn't work at all.

Yeah same with me. I've tried different sources and none of them will cast.

**Comment by transferableethos** [comment]
> So in the end I downloaded BitLord and use magnet.dl to get any of the torrents I need. Allows you to stream or download anything to your device and casting while streaming works like a charm 

How do you do this?
